### DISCUSSION TITLE
More metadata on preview modal

### DISCUSSION THREAD

**Discussion by miagg** [oneduo/nova-file-manager#128] [Q&A] [answered]
Hello,
This package looks awesome and I'm sure it will become the turn-key solution for file manager within nova. 👏

Is it possible to add more meta for the file in the preview modal? Particularly resolution!
Also, I think it misses a "select" action but that's for another discussion.

Thanks!

![modal](https://user-images.githubusercontent.com/6439071/214497764-77ba8ce6-817e-434f-91f5-66c594c4ff30.png)

**Comment by Rezrazi** [comment, marked as answer]
To add more metadata, you may extend the `\Oneduo\NovaFileManager\Entities\Image` class and override the mapping in the config `config('entities.image')`, that should do the trick I believe.

For the addition of a select button, that should be possible, be sure to create an issue and we would then mark it and consider it for future developments.

**Reply by miagg** [reply]
Hi @Rezrazi ,
I think the vue component only renders size, type and last modified date independent of what meta exist.
The Image entity for example does have resolution by default but it does not appear on the preview modal.

**Reply by miagg** [reply]
Let me know if I should open an issue

**Reply by Rezrazi** [reply]
The Vue components loops on all available meta entries

**Reply by miagg** [reply]
I found the cause. Fixed in PR #133
